src/handlers/generate-erc20-permit.ts

```diff
@@ -74,10 +74,14 @@ export async function generateErc20PermitSignature(
     throw new Error("Provider is not defined");
   }
 
-  const { privateKey } = await decryptKeys(_evmPrivateEncrypted);
-  if (!privateKey) {
-    const errorMessage = "Private key is not defined";
-    _logger.fatal(errorMessage);
+  let privateKey = '';
```

Comment:
Empty string seems not the best

Reply:
fixed https://github.com/ubiquibot/permit-generation/pull/59/commits/4f471f3b91caeb30e9454499d5506fa4a2824de7